src/components/TextField.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
I think this component is [premature optimization](https://stackify.com/premature-optimization-evil/). It just abstracts away setting margins and text-align (which could be set in the parent component in fact).

If you want to reuse this component you'll soon end up adding new props until you end up with another kind of styled component. So I'd vote for removing it and just use separate styled components for the headline and subline